src/components/Accordion/components/Compact.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { Colors } from '../../../essentials';
+import { Box } from '../../Box/Box';
+import { Headline } from '../../Headline/Headline';
+import { Header } from './Header';
+import { ChevronUp } from './ChevronUp';
+import { ChevronDown } from './ChevronDown';
+import { Description } from './Description';
+import { PanelButton } from './PanelButton';
+
+interface Props {
+    title: string;
+    description?: string;
+    expanded?: boolean;
+    children?: ReactNode;
+}
+
+const PanelHeader = styled(Header).attrs({ px: '0' })`
+    // @ts-ignore
+    &:hover ${Headline} {
+        color: ${Colors.ACTION_BLUE_1000};
+    }
+
+    &:hover ${ChevronDown} {
+        color: ${Colors.ACTION_BLUE_1000};
+    }
+
+    &:hover ${ChevronUp} {
+        color: ${Colors.ACTION_BLUE_1000};
+    }
+`;
+
+const Compact = ({ title, description, expanded = false, children }: Props) => {
+    const [isOpen, setIsOpen] = useState<boolean>(expanded);
+
+    return (
+        <>
+            <PanelHeader onClick={() => setIsOpen(!isOpen)}>
+                <Box display="flex" flexDirection="column" maxWidth="33%">
+                    <Headline as="h4" mr="3">
+                        {title}
+                    </Headline>
+                    {isOpen && <Description mt="1" description={description} />}
+                </Box>
+                {!isOpen && <Description description={description} />}
+                <PanelButton isOpen={isOpen} />
+            </PanelHeader>
+            {isOpen && <Box mb="5">{children}</Box>}
+        </>
+    );
+};
+export default Compact;
```

Comment:
We usually don't have default exports in this project. There are a few reasons why named exports are preferred, mainly for maintainability down the road.

- [ ] Please use a named export for this component

Reply:
Done